tests/integration_tests/test_sql_task.py

```diff
@@ -333,6 +333,89 @@ def test_sql_access_request_task(db, policy, integration_postgres_config) -> Non
         > 0
     )
 
+# todo- add mssql test
```

Comment:
is this a follow-up TODO? does it need a ticket?

Reply:
I'll go ahead and add this now!